lib/app/Readme.md

```diff
@@ -8,12 +8,20 @@ In your configuration, you can enable an app using something like this:
 
 ```yaml
 apps:
-    # Replace appName with the name of the app you wish to enable
-    appName:
-        responseCache: 180
-        disabled: false
+  # Replace appName with the name of the app you wish to enable
+  appName:
+    responseCache: 180
+    disabled: false
+    sharedData:
```

Comment:
The big question is: what defines configuration? You can really send anything you like, and it doesn't have to fall under the name "configuration". It's just app-specific "stuff".